e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -402,6 +403,51 @@ describe("scenarios > dashboard", () => {
         popover().findByText("Edit question").should("be.visible").click();
         cy.findByRole("button", { name: "Visualize" }).should("be.visible");
       });
+
+      it("should allow navigating to the model editor directly from a dashboard card", () => {
+        createQuestionAndDashboard({
+          questionDetails: {
+            name: "Orders model",
+            type: "model",
+            query: {
+              "source-table": ORDERS_ID,
+            },
+          },
+          dashboardDetails: {
+            name: "Dashboard",
+          },
+        }).then(({ body: { dashboard_id, ...rest } }) => {
+          visitDashboard(dashboard_id);
+        });
+
+        showDashboardCardActions();
+        getDashboardCardMenu().click();
+        popover().findByText("Edit model").should("be.visible").click();
+        cy.location("pathname").should("match", /\/model\/.*\/query/);
```

Comment:
`createQuestionAndDashboard` will resolve with `questionId` (in `...rest`) - we can use it to exactly match the pathname